Bloon/Features/ModTools/Models/ModEvents.cs

```diff
@@ -0,0 +1,29 @@
+namespace Bloon.Features.ModTools
+{
+    using System;
+    using System.ComponentModel.DataAnnotations;
+    using System.ComponentModel.DataAnnotations.Schema;
+
+    [Table("mod_events")]
```

Comment:
Singular

Reply:
w0w yes okay i shall fix this very front end related issue ty for point it out